src/app/event_handlers.py

```diff
@@ -11,10 +19,29 @@
 @webhook_handler.add(MessageEvent, message=TextMessage)
 def handle_text_message_event(event: MessageEvent):
     reply_token = event.reply_token
+    message = event.message.text
 
-    # default response
-    message = HelpMessageCreator().generate()
-    line_api.reply_message(reply_token, message)
+    if product_code := UqProductCodeParser(message).get_product_code_from_url():
```

Comment:
@brianhu-tw We need to change this line as the `UqProductCodeParser` interface is different.